packages/grafana-ui/src/components/Stack/Stack.tsx

```diff
@@ -0,0 +1,30 @@
+import { css } from '@emotion/css';
+import React, { CSSProperties, useCallback } from 'react';
+
+import { GrafanaTheme2, ThemeSpacingTokens } from '@grafana/data';
+
+import { useStyles2 } from '../../themes';
+
+interface StackProps {
+  direction?: CSSProperties['flexDirection'];
+  alignItems?: CSSProperties['alignItems'];
+  wrap?: boolean;
+  gap?: ThemeSpacingTokens;
+  flexGrow?: CSSProperties['flexGrow'];
+}
+
+export const Stack = ({ children, ...props }: React.PropsWithChildren<StackProps>) => {
+  const styles = useStyles2(useCallback((theme) => getStyles(theme, props), [props]));
+  return <div className={styles.root}>{children}</div>;
+};
+
+const getStyles = (theme: GrafanaTheme2, props: StackProps) => ({
+  root: css({
+    display: 'flex',
+    flexDirection: props.direction ?? 'row',
+    flexWrap: props.wrap ?? true ? 'wrap' : undefined,
+    alignItems: props.alignItems,
+    gap: theme.spacing(props.gap ?? 2),
+    flexGrow: props.flexGrow,
```

Comment:
I found this used in one place. Since this is a layout component, should we remove this, and instead ask them to either use `Flex` or `Grid` to solve the problem?